client/components/ConfigureActiveRuns/index.jsx

```diff
@@ -244,16 +244,22 @@ class ConfigureActiveRuns extends Component {
                 onChange={this.handleVersionChange}
                 as="select"
             >
-                {testVersions.map(version => {
-                    return (
-                        <option key={version.id} value={version.id}>
-                            {version.git_hash.slice(0, 7) +
-                                ' - ' +
-                                version.git_commit_msg.slice(0, 30) +
-                                '...'}
-                        </option>
-                    );
-                })}
+                {testVersions
+                    .filter(
```

Comment:
Same question but place in front end code. Why are we filtering here, just curious

Reply:
Because now we are passing all the test versions. The other option was to add a second endpoint but we decided against it.